pkg/endpoint/events.go

```diff
@@ -214,20 +215,49 @@ func (ev *EndpointPolicyBandwidthEvent) Handle(res chan interface{}) {
 		e.unlock()
 	}()
 
-	bandwidthEgress, err := ev.annoCB(e.K8sNamespace, e.K8sPodName)
+	bandwidthEgress, priority, err := ev.annoCB(e.K8sNamespace, e.K8sPodName)
 	if err != nil {
 		res <- &EndpointRegenerationResult{
 			err: err,
 		}
 		return
 	}
+	bwmUpdateNeeded := false
 	if bandwidthEgress != "" {
 		bps, err = bandwidth.GetBytesPerSec(bandwidthEgress)
-		if err == nil {
-			ev.bwm.UpdateBandwidthLimit(e.ID, bps)
-		} else {
-			e.getLogger().WithError(err).Debugf("failed to parse bandwidth limit %q", bandwidthEgress)
+	}
+	if err != nil {
+		e.getLogger().WithError(err).Debugf("failed to parse bandwidth limit %q", bandwidthEgress)
+	} else {
+		bwmUpdateNeeded = true
```

Comment:
if egress bw not set , the bwmUpdateNeeded is still be true ?

Reply:
If it was set previously and now unset, then yes. @l1b0k could you double check if that works?